sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb

```diff
@@ -75,6 +75,10 @@
     "region = session.boto_region_name\n",
```

Comment:
<p>If we use an existed endpoint, <code>instance_type</code>&nbsp;and&nbsp;<code>instance_count</code> is no longer needed in the ModelConfig. Please add some explanation here and also provide that as an option.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Thanks, improved that part.